site/app/templates/grading/electronic/Details.twig

```diff
@@ -50,9 +50,29 @@
                     {{ anon_mode ? "Disable Anonymous Mode" : "Enable Anonymous Mode" }}
                 </button>
                 {% endif %}
+                {% if grade_inquiry_only_button %}
+                <button id="inquiryButton" class="btn btn-default" onclick="change_inquiry()">
```

Comment:
Lets just switch the order to class, id, onclick. Maintaining same style as above codes :)